### ISSUE TITLE
rac installation, nodelist

### ISSUE THREAD

**Issue by stelb** [oravirt/ansible-oracle#127]
I tried to install a RAC DB, which failed with some obscure error after successful GI Installation:

[FATAL] [DBT-14505] Oracle RAC software is not present on nodes ([db01n1.dev.int.company.com, db01n2.dev.int.company.com]) selected for configuration.
   ACTION: Install Oracle RAC software on all the selected nodes before doing database configuration.

i did not find any mention of this error.

Now i realized, that the nodes are listed with FQDN. I rmeoved the domain and it worked.

the template loops over the current hostgroup in the inventory. the hosts given with FQDN

I cannot use just db01n1 because we have lots of these, for different customers..
So there should be a filter to remove the domain I think.

Maybe i did something wrong, but i have no clue what..

**Comment by oravirt** [comment]
What does your inventory look like?

**Comment by stelb** [comment]
```
➜  ao git:(master) ✗ cat inventory/devrac
[devrac]
db01n1.dev.int.nexinto.com
db01n2.dev.int.nexinto.com
```

**Comment by stelb** [comment]
I have changed 
```
host
```
to
```
host.split('.')[0]
```

in the template

**Comment by stelb** [comment]
ah.. did not see group_vars question..
```
➜  ao git:(master) ✗ ls group_vars/devrac
asm.yml           installmedia.yml  monitoring.yml    ora_databases.yml ora_pdbs.yml      passwords.yml
gi.yml            misc.yml          ora.yml           ora_dbhome.yml    os.yml

➜  ao git:(master) ✗ cat group_vars/devrac/{gi,ora_*}.yml | grep -v ^#
---
oracle_install_version_gi: 12.2.0.1
oracle_asm_init_dg: si_data

configure_cluster: true
oracle_gi_cluster_type: standard
oracle_scan: scan-primary.dev.int.compan.com
oracle_vip: -vip
oracle_gi_nic_pub: eth1
oracle_gi_nic_priv: eth3
oracle_scan_port: 1521
---
oracle_databases:
      - home: ee-12.2.0.1
        oracle_db_name: CDBDIGI
        oracle_db_type: RAC
        is_container: True
        storage_type: ASM
        oracle_db_mem_totalmb: 4096
        oracle_database_type: OLTP
        redolog_size: 100M
        datafile_dest: SI_DATA
        recoveryfile_dest: +SI_DATA
        state: absent
        init_parameters:
          - { name: db_recovery_file_dest_size, value: 10G, state: present, scope: both }
          - { name: db_recovery_file_dest, value: '+SI_DATA', state: present, scope: both }
          #- { name: pga_aggregate_target, value: 500M, state: present, scope: both }
          - { name: _parallel_statement_queuing, value: "false", state: present, scope: both }
          - { name: sga_target, value: 4G, state: present, scope: spfile }
          - { name: sga_max_size, value: 4G, state: present, scope: spfile }
          - { name: open_cursors, value: 400, state: present, scope: both }
          - { name: processes, value: 200, state: present, scope: spfile }

```

**Comment by oravirt** [comment]
No problem - I also misread the problem slightly, thought you had issues with the sw installation at first.

I've reproduced the problem, so a fix is coming up.

**Comment by oravirt** [comment]
Should be fixed with ce2307e8fa9e0322ea6d68edb4a5d22a90bdc3ec

**Comment by stelb** [comment]
works! thank you
